proto/buf/alpha/registry/v1alpha1/repository_branch.proto

```diff
@@ -33,9 +33,12 @@ message RepositoryBranch {
   google.protobuf.Timestamp last_update_time = 5;
   // The ID of the user who updated the branch.
   string last_update_user_id = 6;
-  // The author name of the most recent associated git commit of the branch. May be null
+  // The author name of the most recent associated git commit of the branch. May be an empty string
   // if the last branch update doesn't have a git commit associated.
   string last_update_git_author_name = 7;
+  // The git commit hash of the most recent associated git commit of the branch. May be an empty string
+  // if the last branch update doesn't have a git commit associated.
+  string last_update_git_commit_hash = 8;
```

Comment:
Do we use the terminology `hash` to refer to git commits anywhere else in our APIs?

Reply:
Yes, it's used in several places, including [GitCommitInformation](https://buf.build/bufbuild/buf/docs/main:buf.alpha.registry.v1alpha1#buf.alpha.registry.v1alpha1.GitCommitInformation) and [GitSyncPoint](https://buf.build/bufbuild/buf/docs/main:buf.alpha.registry.v1alpha1#buf.alpha.registry.v1alpha1.GitSyncPoint).
Do you see this being an issue?

Reply:
OK